plugin/codeium.vim

```diff
@@ -60,7 +60,19 @@ if !get(g:, 'codeium_disable_bindings')
 endif
 
 call s:SetStyle()
-call timer_start(0, function('codeium#server#Start'))
+
+" Run codeium server if not already started
+" FIXME: share this with command.vim
+function! s:start_language_server()
```

Comment:
I believe if you rename this function to not start with `s:` you can access it from other files. 

Rename it to `CodeiumStartLanguageServer`

Reply:
done